src/main/java/Controller.java

```diff
@@ -0,0 +1,53 @@
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class Controller {
+
+    public static void main(String[] args) {
+        /**
+         * We create the input getter, and scheduler. Note that scheduler has addFilters() method, which we use
+         * according to User Input later in this method
+         */
+        CommandLineInterface inputGetter = new CommandLineInterface();
```

Comment:
this is violating clean architecture, as the controller shouldn't be relying on the user interface